dags/js/IO/file-utils.js

```diff
@@ -1,12 +1,11 @@
 const fs = require('fs');
+const util = require('util');
 
 
 module.exports.deleteFile = async (filePath) => {
-  if (fs.existsSync(filePath)) {
-    await fs.unlink(filePath, (error) => {
-      if (error) throw error;
-      console.log('successfully deleted ' + filePath);
-    });
+  if (await util.promisify(fs.exists)(filePath)) {
+    await util.promisify(fs.unlink)(filePath);
```

Comment:
Ditto here, by using `unlinkSync`

Reply:
Blocking IO should not be a standard for production code?

Reply:
But `deleteFile` is async so that should be ok then?

Reply:
See other comments chain